cmd/server.go

```diff
@@ -167,7 +169,7 @@ func runServer(cmd *cobra.Command, args []string) error {
 	openApiSpec := generateOpenApiSpec()
 	router.Get("/api-docs", func(w http.ResponseWriter, r *http.Request) {
 		if _, responseFailed := w.Write(openApiSpec); responseFailed != nil {
-			log.Printf("failed responding to /api-docs: %v", responseFailed)
+			log.Error("failed responding to /api-docs:", slog.Any("error", responseFailed))
```

Comment:
Same, consider a common field names constants so we don't have to search replace in many places.